tests/functional/conftest.py

```diff
@@ -22,7 +22,7 @@ def pytest_configure():
     conf_vars = {'__file__': ''}
     with open(settings_path) as settings_file:
         old_path = sys.path
-        sys.path = [TESTDIR]
+        sys.path = [TESTDIR] + [p for p in old_path if 'python' in p]
```

Comment:
Since i add a new conf in settings and require `from cryptography.fernet import Fernet`.So cryptography and it's dependence modules will need when execute `exec(output, conf_vars)`.If i don't add python related path to sys.path.It will hit ModuleNotFoundError like this [https://gitlab.corp.redhat.com/product-security/dev/openlcs/-/jobs/1463262](url).I also want to know why replace sys.path to [TESTDIR] before execute `exec(output, conf_vars)`.Thanks.

Reply:
Added docstring in code to clarify.